DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -574,7 +619,22 @@ void TopMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
   }
   if ( muons.size() < nmuons_ ) return;
 
+    //george
+
+  if (nmuons_>1){
+    TLorentzVector mu1,mu2;
+    mu1.SetPtEtaPhiM(muons.at(0).pt(),muons.at(0).eta(),muons.at(0).phi(),0.105);
+    mu2.SetPtEtaPhiM(muons.at(1).pt(),muons.at(1).eta(),muons.at(1).phi(),0.105);
+    mll=(mu1+mu2).M();
```

Comment:
mll= (muons[0]+muons[1]).M() should be the same but without all the variable copies? You could also consider using M2() and then cutting on the mass squared value below - it would be more efficient.